geomstats/datasets/graph_data_preparation.py

```diff
@@ -93,4 +93,4 @@ def _walk(self, index, walk_length):
             count_index = self.edges[count_index][random.randint(
                 0, len(self.edges[count_index]) - 1)]
             path.append(count_index)
-        return gs.array(path, dtype=int)
+        return gs.array(path).astype(int)
```

Comment:
Why this change? Maybe also use `gs.int32` so that the example will be easier to port to tf and pytorch later?